server/lib/buff_server/packages.ex

```diff
@@ -50,7 +53,10 @@ defmodule BuffServer.Packages do
       {:error, %Ecto.Changeset{}}
 
   """
-  def create_package(attrs \\ %{}) do
+  def create_package(attrs \\ %{}, artifact_binary) do
```

Comment:
add a bang to this function.. it can now throw.

Reply:
👍